src/searchForm/datePicker/datePickerTopBar/DatePickerTopBar.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import CalendarIcon from 'icon/calendarIcon'
+import Text from 'text/Text'
+import Button, { ButtonStatus } from 'button'
+import cc from 'classcat'
+import ChevronIcon from 'icon/chevronIcon'
+
+export interface DatePickerTopBarProps {
+  readonly title: string
+  readonly className?: Classcat.Class
+  readonly isClickable?: boolean
+}
+
+export const DatePickerTopBar = (props: DatePickerTopBarProps) => (
+  <div className={cc(['kirk-datePickerTopBar-row', props.className])}>
```

Comment:
why is it not an kirk item with a left addon  ?

Reply:
It's an Item now